utils/build/styled.d.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import 'styled-components';
-import { IGardenTheme } from '../../packages/theming';
+import type { IGardenTheme } from '../../packages/theming/src/index';
```

Comment:
Pointing to the source rather than the `./dist` output allows the `lint` job (`npm exec tsc`) to run in parallel with `build`.